packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -386,9 +389,23 @@ function updateOutputPort(node: WorkflowNode<any> | null, workflowOutput: Workfl
 
 const openDrilldown = (node: WorkflowNode<any>) => {
 	currentActiveNode.value = node;
+	startTime = Date.now();
 	dialogIsOpened.value = true;
 };
 
+const closeDrilldown = async () => {
+	dialogIsOpened.value = false;
+	const timeSpent: number = Date.now() - startTime;
+	await EventService.create(
+		EventType.OperatorDrilldownTiming,
+		useProjects().activeProject.value?.id,
+		JSON.stringify({
+			node: currentActiveNode.value?.displayName,
+			timeSpent
+		})
+	);
+};
```

Comment:
I am not sure what this is trying to measure, but note `dispalyName` is not unique as you can have several operators of the same type.

Reply:
We're just looking to see how long people are in each of the drilldowns... so I _think_ just using displayName here should be ok but we could log out more info if you think thats useful?

Reply:
I don't know either, I suppose we can always change if needed. 

If in doubt we can serialize the entire `currentActiveNode.value`